monetio/models/_camx_mm.py

```diff
@@ -448,7 +480,11 @@ def _calc_midlayer_height_msl(dset):
         alt_agl_m_mid = dset["alt_agl_m_mid"]
     else:
         alt_agl_m_mid = _calc_midlayer_height_agl(dset)
```

Comment:
```suggestion
        alt_agl_m_mid, _ = _calc_midlayer_height_agl(dset)
```
seems like this needs to be updated?

Reply:
Yes